### ISSUE TITLE
Conformance test for bindings list operation does not invoke ListOperation

### ISSUE THREAD

**Issue by CodeMonkeyLeet** [dapr/components-contrib#1051]
## Expected Behavior

Bindings tests that specify the `list` operation should invoke the List operation.

## Actual Behavior

The `list` operation ends up invoking the Get operation on line 195:

https://github.com/dapr/components-contrib/blob/dc7ee55e2ab7006f9d216f677f3a05c25d7e03fd/tests/conformance/bindings/bindings.go#L191-L199

Making the one line code fix causes the only test to use the `list` operation to fail (azure.blobstorage) with:

```bash
=== RUN   TestBindingsConformance/azure.blobstorage/list
    bindings.go:199: 
        	Error Trace:	bindings.go:199
        	Error:      	Received unexpected error:
        	            	json: cannot unmarshal array into Go value of type blobstorage.listPayload
        	Test:       	TestBindingsConformance/azure.blobstorage/list
        	Messages:   	expected no error invoking output binding
```

## Steps to Reproduce the Problem

Code inspection, run conformance tests using the conformance.yml workflow or [manual instructions](https://github.com/dapr/components-contrib/blob/master/tests/conformance/README.md)

## Release Note 

Test fix, should not require a release note.

**Comment by CodeMonkeyLeet** [comment]
Bug remains active pending a fix.

**Comment by rainfd** [comment]
already fix in #1317
